src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,99 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImageURL] = useState('');
+  const [imdbUrl, setImdbURL] = useState('');
+  const [imdbId, setImdbID] = useState('');
```

Comment:
Create only one object as a state with the required keys and values
```suggestion
  const [movie, setMovie] = useState({
    title: '',
    ..........
  })
```